auto-generated/cluster/stone-prd-rh01/tenants/tekton-ecosystem-tenant/appstudio.redhat.com_v1beta1_integrationtestscenario_ecosystem-images-enterprise-contract.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: appstudio.redhat.com/v1beta1
+kind: IntegrationTestScenario
+metadata:
+  name: ecosystem-images-enterprise-contract
+  namespace: tekton-ecosystem-tenant
+spec:
+  application: ecosystem-images
```

Comment:
@ernesgonzalez33 I have a query here
under one application we can have multiple component so the question is does integration tests are same for all component under one application
basically ask is integration tests are across application ?

Reply:
Yes. Integration tests cover the entire application